osidb/models.py

```diff
@@ -1536,6 +1537,9 @@ def __init__(self, *args, **kwargs):
 
     # internal primary key
     uuid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    # A unique ID of this snippet as it is defined in the external data source where the
+    # snippet was collected from.
+    external_id = models.CharField(max_length=200)
```

Comment:
The first commit should also contain the respective migration, which is currently included later. Also, I am not whether it is ok from the Django perspective to add a mandatory field into already existing model without a default value/null.